common/governments/civics/btr_origins.txt

```diff
@@ -142,14 +142,19 @@ origin_omega = {
 	} 
 }
 
-origin_parnack_regime = { 
+btr_origin_parnkreg = { 
```

Comment:
Just as a cautionary remark, I recall more files referencing origin_parnack_regime so if you could double check that, it would be really nice.

Reply:
All clean